### PR TITLE
Create task 2

### PR DESCRIPTION
**Pull request by Garrett-Donowan**

### CODE REVIEW

File: Tasks/Gorbachev/Task2.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    h_1 = {1: 'один', 2:'два', 3:'три' ,4:'четыре' ,5:'пять' ,6:'шесть' ,7:'семь',
+            8:'восемь' ,9:'девять' ,10:'десять' ,11:'одинадцать' ,12:'двенадцать'}
+    h_2_12 = {1:'второго' ,2:'третьего' ,3:'четвёртого' ,4:'пятого' ,5:'шестого' ,6:'седьмого',
+            7:'восьмого' ,8:'девятого' ,9:'десятого' ,10:'одиннадцатого' ,11:'двенадцатого' ,12:'первого'}
+    m_2_12 = {1: 'одной', 2: 'двух', 3: 'трех', 4: 'четырех', 5: 'пяти', 6: 'шести', 7: 'семи', 8: 'восьми', 9: 'девяти', 10: 'десяти',
+         11: 'одиннадцати', 12: 'двенадцати', 13: 'тринадцати', 14: 'четырнадцати', 15: 'пятнадцати'}
+    m_1 = {1: 'одна', 2: 'две', 3: 'три', 4: 'четыре', 5: 'пять', 6: 'шесть', 7: 'семь', 8: 'восемь', 9: 'девять'}
+    m_2 = {10: 'десять', 20: 'двадцать', 30: 'тридцать', 40: 'сорок', 50: 'пятьдесят'}
+    m_3 = {11: 'одиннадцать', 12: 'двенадцать', 13: 'тринадцать', 14: 'четырнадцать', 15: 'пятнадцать', 16: 'шестнадцать',
+         17: 'семнадцать', 18: 'восемнадцать', 19: 'девятнадцать'}
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gorbachev/Task2.py:15]: a very good data structure

File: Tasks/Gorbachev/Task2.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ 
+def user_time():  # enter user time
+    k = False
+    while k == False:
+        print(f"Please enter valid time in format HH:MM")
+        u_time = input()
+        u_hour, u_minutes = u_time.replace(" ", "").replace("-",":").replace("_",  ":").split(":")
+        u_hour = int(u_hour) - 12
+        u_minutes = int(u_minutes)
+        if (0 <= u_hour) and (u_hour <= 12) and (0 <= u_minutes) and (u_minutes <= 60):
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gorbachev/Task2.py:57]: I would suggest to check for the format as well

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Very good! Both data structure and the main logic are very thoughtful and clean. The functions are decently implemented as well. Your score is 9.5.

File: Tasks/Gorbachev/Task2.py (hunk 2 above)
Comment (Garrett-Donowan) [reply Tasks/Gorbachev/Task2.py:57]: yep. seems my fault. miss this check.

**Pull request merged**